analysisOnData/runAnalysisOnWJetsMC.py

```diff
@@ -91,7 +90,7 @@ def RDFprocessWJetsMC(fvec, outputDir, sample, xsec, fileSF, ncores, pretendJob,
                 p.branch(nodeToStart = 'defs', nodeToEnd = 'defsAC', modules = steps)
                 p.branch(nodeToStart = 'defsAC', nodeToEnd = '{}/templatesAC_{}/{}Vars'.format('WToMu', region, s), modules = [ROOT.templateBuilder(wtomu_cut, var_weight,vars_vec,variations[1], 3)])
                 #reco templates for out of acceptance events
-                wtomu_cut+= "&& GenV_preFSR_qt>32. && GenV_preFSR_yabs>2.4"
```

Comment:
@sroychow

Reply:
this line puzzles me

Reply:
@emanca This was in the commented out section in the Wjets config[[here](https://github.com/emanca/wproperties-analysis/blob/master/analysisOnData/runAnalysisOnWJetsMC.py#L94)]. So I changed it to be consistent with the branch names in the private MC.